docs/modules/ROOT/pages/erc20.adoc

```diff
@@ -88,15 +84,16 @@ Use cases go from medium of exchange currency to voting rights, staking, and mor
 
 Considering that the constructor method looks like this:
 
-[,python]
+[,cairo]
 ----
 func constructor(
-    name: felt,               # Token name as Cairo short string
-    symbol: felt,             # Token symbol as Cairo short string
-    decimals: felt            # Token decimals (usually 18)
-    initial_supply: Uint256,  # Amount to be minted
-    recipient: felt           # Address where to send initial supply to
-):
+    name: felt,               // Token name as Cairo short string
+    symbol: felt,             // Token symbol as Cairo short string
+    decimals: felt            // Token decimals (usually 18)
+    initial_supply: Uint256,  // Amount to be minted
+    recipient: felt           // Address where to send initial supply to
+) {
+}
```

Comment:
```suggestion
) {}
```

Reply:
This looks it should work, but the compiler will yell at you with:

```
Unexpected token Token('RBRACE', '}'). Expected: .
```

The final `}` must be on its own line. See this discussion [https://github.com/OpenZeppelin/cairo-contracts/pull/449#discussion_r963050900](https://github.com/OpenZeppelin/cairo-contracts/pull/449#discussion_r963050900)